pkg/util/grpc_test.go

```diff
@@ -60,7 +69,12 @@ func (m *mockStream) CloseSend() error {
 
 func (m *mockStream) Recv() (string, error) {
 	if len(m.recvErrors) == 0 {
-		// Block forever.
+		// Block forever, unless the context is canceled (if provided).
```

Comment:
Note to reviewers: fixed a goroutine leak which was causing `TestVariableTicker` to fail.